### PR TITLE
I have added the desired HTML functionality to the ratings page

### PR DESCRIPTION
**Pull request by andreimastin**
I have added the buttons and styling to the HTML for the rating page. I have also marked the spots where the movie field data must be added, the buttons also have stubbed commands

### CODE REVIEW

File: Template.html [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+            border: none !important;
+        }
+
+        td {
+            border: none !important;
+        }
+
+        .input {
+            height: 30px !important;
+        }
```
Comment (JaadVdW) [inline Template.html:43]: Perhaps clean up unnecessary spacing here

File: rating.html [hunk 2]
```diff
@@ -1,17 +1,108 @@
...
+
+                     <!--The OnClick() values for the buttons MUST be changed to accurate ones-->
+                     <tr>
+                         <td>
+                             <input type="button" class="button is-Disliked" value="Dislike" onClick="isDisliked()" style="font-size: 30px; border-color: black; font-weight: bold;color:white; background-color: red; width: 300px">
+
+                         </td>
+
+                         <td>
+                             <input type="button" class="button is-Liked" value="Like" onClick="isLiked()" style="font-size:30px;font-weight:bold;border-color: black; color:white; background-color:green; width:300px">
```
Comment (JaadVdW) [inline rating.html]: Good catch on button sizing to be the same

File: rating.html [hunk 3]
```diff
@@ -1,17 +1,108 @@
...
+    <!--Put  Tab Title Here-->
+    <title>Rate Movies</title>
+    <!--Set General Style Settings-->
+    <style>
 
-    <div class="displayMovie">
-        <h2> This is where the movie to choose will be inserted.</h2>
-        <body>
-        </body>
-    </div>
```
Comment (JaadVdW) [inline rating.html]: Removing this should be fine as long as we are displaying similar text on the page for the user, which I see below

File: rating.html [hunk 4]
```diff
@@ -1,17 +1,108 @@
...
+            -moz-transform: translateX(-50%) translateY(-50%);
+            -webkit-transform: translateX(-50%) translateY(-50%);
+            transform: translateX(-50%) translateY(-50%);
+            width: 80;
+            margin: auto;
+        }
 
-    <div class="ratingMechanism">
-        <h2> This section will contain the yes and no buttons to choose whether the user is interested in a movie or not.</h2>
-    </div>
```
Comment (JaadVdW) [inline rating.html]: Same here, removing is fine as long as we have this heading elsewhere on the page in proper format, seen at line 56

File: Template.html (hunk 1 above)
Comment (andreimastin) [reply Template.html:43]: I like the spacing for clarity

File: rating.html (hunk 3 above)
Comment (andreimastin) [reply rating.html]: I removed it, it should be good now.

**Comment by andreimastin** [comment]
I have conducted an integration test.  to see if the button triggers the endpoints.

**Pull request merged**